### DISCUSSION TITLE
Networking and connecting to node

### DISCUSSION THREAD

**Discussion by RayOkaah** [trustwallet/wallet-core#1711] [Q&A]
Hello.
I unfortunately couldn't find any resource on connecting trust wallet core to the blockchain and how that works. Does anyone care to pint me in the right direction?
Thanks.

**Comment by optout21** [comment]
Wallet core library does not contain code for connecting to blockchains, that has to be implemented separately, for each blockchain.  A good starting point is the node/RPC documentation of the desired blockchain.  Operations needed: bringing transactions, balance, UTXos, broadcasting new transaction.

**Comment by RayOkaah** [comment]
Okay. Thanks.
